internal/linker/index.js

```diff
@@ -1,4 +1,5 @@
-/* THIS FILE GENERATED FROM .ts; see BUILD.bazel */ /* clang-format off */var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+/* THIS FILE GENERATED FROM .ts; see BUILD.bazel */ /* clang-format off */"use strict";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```

Comment:
This file changed quite a bit, but it seems like it really only a) dropped (all?) the comments  and b) dropped the UMD def which I guess was pointless?

is it the generated tsconfig.json vs previous ts_library config that changed this?

Reply:
sorry, I should have explained in the commit message - yes it's just as you say, now that it uses `ts_project` we have more control so we don't bother with UMD, and as Greg is adding a lot of comments in the linker right now I figured we could save the 1% size on the builtin tgz.

https://github.com/bazelbuild/rules_nodejs/pull/1811/commits/ee3e065b763b7eb59777cff6217909f63977c31f#diff-28cc3d70512445827e84e7bd2f4adb49R31-R32

Reply:
Yeah that makes sense 👍 

I assume the other checked-in generated file is the same changes? That one was also renamed so you can't really see the diff...

Reply:
right, identical changes. the rename is unfortunate but I couldn't find a less disruptive way to switch to ts_project, since it predeclares/labels the output js and that causes collision with input artifact